src/components/ResultCard/ResultCard.tsx

```diff
@@ -5,66 +5,31 @@ import QuizContext from '../CioQuiz/context';
 
 interface ResultCardProps {
   result: Partial<BrowseResultData>;
-  addToCartCallback: (clickedResult: Partial<BrowseResultData>) => any;
-  clickItemCallback?: (clickedResult: Partial<BrowseResultData>) => any;
   salePriceKey?: string;
   regularPriceKey?: string;
   resultPosition: number;
 }
 
 export default function ResultCard(props: ResultCardProps) {
-  const {
-    result,
-    addToCartCallback,
-    clickItemCallback: customClickItemCallback,
-    salePriceKey,
-    regularPriceKey,
-    resultPosition,
-  } = props;
-  const { resultsResponse, cioClient } = useContext(QuizContext);
+  const { result, salePriceKey, regularPriceKey, resultPosition } = props;
+  const { resultClickHandler, customClickItemCallback } = useContext(QuizContext);
   const salePrice = salePriceKey && result?.data?.[salePriceKey];
   const regularPrice = regularPriceKey && result?.data?.[regularPriceKey];
 
-  const clickItemCallback = () => {
-    /* eslint-disable @typescript-eslint/naming-convention */
-    if (resultsResponse && resultsResponse.request && resultsResponse.response) {
-      const {
-        quiz_id,
-        quiz_session_id,
-        quiz_version_id,
-        result_id,
-        request: { section, page, num_results_per_page },
-        response: { total_num_results },
-      } = resultsResponse;
-      /* eslint-enable @typescript-eslint/naming-convention */
-
-      cioClient?.tracker.trackQuizResultClick({
-        quiz_id,
-        quiz_version_id,
-        quiz_session_id,
-        item_id: result.data?.id,
-        item_name: result?.value,
-        section,
-        result_count: total_num_results,
-        result_page: page,
-        result_id,
-        result_position_on_page: resultPosition,
-        num_results_per_page,
-      });
-    }
-
-    if (customClickItemCallback && typeof customClickItemCallback === 'function') {
-      customClickItemCallback(result);
+  const clickHandler = (item: Partial<BrowseResultData>) => {
+    if (resultClickHandler) {
+      resultClickHandler(item, resultPosition);
```

Comment:
`resultPosition` is already in scope, so we are grabbing it from upper scope rather than passing it to `clickHandler` as an argument. Shall we follow the same pattern for `item` aka `result`?